lib/ActivityStreams.js

```diff
@@ -59,6 +61,7 @@ ActivityStreams.prototype = {
     // if the function is async, the worker might not be there yet, or might have already disappeared
     try {
       worker.port.emit("addon-to-content", action);
+      this._logPerformance(action.type, worker);
```

Comment:
Why do we need the indirection? Can we call `this._perfMeter.log(worker.tab, action.type)` directly?